src/partials/editor.html

```diff
@@ -187,6 +187,16 @@ <h6 ng-show="ctrl.panel.locationData === 'table' || ctrl.panel.locationData ===
       <input type="text" class="input-small gf-form-input width-10" ng-model="ctrl.panel.esMetric" ng-change="ctrl.refresh()" ng-model-onblur
       />
     </div>
+    <div class="gf-form" ng-show="ctrl.panel.locationData === 'geohash'">
+      <gf-form-switch class="gf-form" ng-show="ctrl.panel.locationData === 'geohash'" label="Apply log Scale" label-class="width-12" checked="ctrl.panel.isLogScale" ng-model="ctrl.panel.isLogScale" on-change="ctrl.render()"></gf-form-switch>
```

Comment:
Why is it `label-class="width-12"` and not `with-10`?

As I can see most of the `input` and `gf-form-switch` are `width-10`

Reply:
The labels are width-12 and the inputs width-10. The input "Hide local names" was, indeed, with the wrong width (10) on it's label, fixing now:
![image](https://user-images.githubusercontent.com/12836540/134644411-0e31b9ee-e423-4c53-9fba-350d8ac62481.png)